neurots/preprocess/utils.py

```diff
@@ -0,0 +1,68 @@
+"""Some utils for preprocessing."""
+
+# Copyright (C) 2022  Blue Brain Project, EPFL
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from collections import defaultdict
+from copy import deepcopy
+from itertools import chain
+
+from neurots.validator import validate_neuron_distribs
+from neurots.validator import validate_neuron_params
+
+_PREPROCESS_FUNCTIONS = {
+    "preprocess": defaultdict(set),
+    "validator": defaultdict(set),
```

Comment:
Minor: Could we have both keys as verbs, i.e. "preprocess" and "validate" for consistency?

Reply:
I changed to `preprocessors` and `validators` because the stored objects are functions. Is it ok for you?

Reply:
Yeah, as far as they are consistent. :+1: